README.md

```diff
@@ -1,3 +1,35 @@
 # 24-03_bioinfo_ws
 
 presentation and resources for NorPreM bioinformatics workshop in March 2024
```

Comment:
I'd go with the full workshop name which is 'A two-day workshop for bioinformaticians and molecular biologists
with focus on the TSO500 pipeline in InPreD'